packages/labs/ssr-react/README.md

```diff
@@ -0,0 +1,72 @@
+# @lit-labs/ssr-react
+
+A package for integrating Lit SSR with React and React frameworks.
+
+## Overview
+
+When server rendering React components that include custom elements authored with Lit, these are shallowly rendered i.e. only the host element's open and closing tags are present in the server rendered HTML generated, not any of its contents.
+
+This package provides tools to integrate with React's JSX transform and element creation such that Lit components will be deeply rendered to include their shadow DOM contents as declarative shadow DOM using [`@lit-labs/ssr`](../ssr/README.md) in the server rendered HTML created by `ReactDOMServer`.
+
+## Usage
+
+This package includes several ways of adding Lit SSR integration to React's base element creation to address different JSX transforms as well as pre-compiled components.
+
+### Using the Classic JSX Transform
+
+The classic JSX transform replaces JSX with `React.createElement` function calls.
+
+This package provides the `patchCreateElement` function which takes the `React.createElement` function and returns an enhanced function that adds a template shadowroot element as a child to the host custom element for any registered Lit components it gets.
+
+```js
+import React from 'react';
+import {patchCreateElement} from `@lit-labs/ssr-react`;
+
+const patchedCreateElement = patchCreateElement(React.creatElement);
```

Comment:
I was thinking users would include this for every JSX file, though that would be wasteful. They could do it once, export the patched/wrapped function as you suggest and import that for every JSX file. Or we could vend an already wrapped function in the package so users can just import that instead of creating their own wrapped version.

In which case the wrap function doesn't really need to be user facing, unless users want to use it for wrapping non-React alternative `createElement` functions.